pkg/skaffold/initializer/init.go

```diff
@@ -74,11 +84,17 @@ func DoInit(ctx context.Context, out io.Writer, c config.Config) error {
 	}
 
 	if !c.Force {
-		if done, err := prompt.WriteSkaffoldConfig(out, pipeline, c.Opts.ConfigurationFile); done {
+		if done, err := prompt.WriteSkaffoldConfig(out, pipeline, generatedManifests, c.Opts.ConfigurationFile); done {
 			return err
 		}
 	}
 
+	for path, manifest := range generatedManifests {
+		if err := ioutil.WriteFile(path, manifest, 0644); err != nil {
```

Comment:
create a descriptive constant for 0644  e.g. `ReadWrite`

Reply:
we use `0644` explicitly several times in the codebase. not sure I want to change all of those in this PR